libs/perun/login/src/lib/login-screen-base/login-screen-base.component.html

```diff
@@ -14,6 +14,9 @@
   <div *ngIf="application === 'user-profile'" class="mt-auto mb-auto">
     <a class="logo-container" [innerHTML]="logo"></a>
   </div>
+  <div *ngIf="application === 'consolidator'" class="mt-auto mb-auto">
+    <div [innerHTML]="logo" class="logo-container"></div>
+  </div>
```

Comment:
It looks like we only handle the publications app diferently, maybe other cases can be merged.